.github/workflows/third_party_notices_check.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Third Party Notices File
+
+on: pull_request
+
+jobs:
+  license-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install generate-license-file
+        run: npm install -g generate-license-file
+      - uses: actions/checkout@v2
+      - run: npm ci
+      - name: Generate license file
+        run: generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES
+      - name: Git diff
```

Comment:
Would it be ok to remove the Git diff and Install generate-license-file step names too?